### PR TITLE
Add overflow to limit multidots

### LINKED ISSUE #1194: Multi-dots limiting solution
node_modules/react-native-calendars/src/calendar/day/multi-dot/index.js line 91

### PR DESCRIPTION
**Pull request by belgamo**
Closes #1194 

image

### CODE REVIEW

File: src/calendar/day/multi-dot/index.js [hunk 1]
```diff
@@ -39,18 +38,38 @@ class Day extends Component {
...
 
   renderDots(marking) {
     const baseDotStyle = [this.style.dot, this.style.visibleDot];
-    if (marking.dots && Array.isArray(marking.dots) && marking.dots.length > 0) {
-      // Filter out dots so that we we process only those items which have key and color property
-      const validDots = marking.dots.filter(d => (d && d.color));
+    if (
+      marking.dots &&
+      Array.isArray(marking.dots) &&
+      marking.dots.length > 0
```
Comment (brunohkbx) [inline src/calendar/day/multi-dot/index.js:55]: I think you can simplify all these conditionals like so: `marking.dots?.length > 0`

Comment (belgamo) [reply src/calendar/day/multi-dot/index.js:55]: Sure. To be honest, linter changed this line haha

**Comment by bryantabaird** [comment]
Thank you for doing this! I ran into this issue today, and noticed that there was no multi-dot truncation when the dot count was very high (over 7 or 8). Anxious to see this enhancement get released

**Comment by Inbal-Tish** [comment]
@belgamo Please remove formatting so I can review only the relevant changes

**Comment by belgamo** [comment]
> @belgamo Please remove formatting so I can review only the relevant changes

Will do! Thanks!

**Review by Fgl — COMMENTED** [review]
Please remove formatting in order to merge these changes

**Pull request closed without merging**